compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1599,5 +1599,8 @@ error.invalid.usage.of.multiple.restricted.vars.in.lock=\
 error.invalid.isolated.qualifier.on.module.no.init.var.decl=\
   an uninitialized module variable declaration cannot be marked as ''isolated''
 
+error.only.simple.variables.are.allowed.to.be.isolated=\
+  only simple variables are allowed to be isolated
```

Comment:
```suggestion
  only a simple variable can be marked as ''isolated''
```

Reply:
Let's update the error code accordingly.

Reply:
Noted